geps/gep-713/index.md

```diff
@@ -3,1155 +3,1192 @@
+
+#### Status needs to be namespaced by implementation
+
+Because an object can be affected by multiple implementations at once, any added status MUST be namespaced by the implementation.
+
+In Gateway API's Route Parent status, `parentRef` plus the controller name have been used for this.
+
+For a policy, something similar can be done, namespacing by the reference to the implementation's controller name.
+
+Namespacing by the originating policy cannot easily be done because the source could be more than one policy object.
+
+#### Creating common data representation patterns
+
+Defining a _common_ pattern for including the details of an _arbitrarily defined_ object, to be included in a library for all possible implementations, is challenging, to say the least.
+
+Structured data cannot be used because there is no way of knowing what the structure will be beforehand. This suggests a need to use unstructured data for representing the main body of the arbitrary policy objects. Practically, this will have to be a string representation of the YAML form (or JSON, equivalently) of the body of the policy object (absent the metadata part of every Kubernetes object).
+
+Metaresources and Policy Attachment does not mandate anything about the design of the object's top level except that it must be a Kubernetes object, so the only possible thing to rely upon here is the presence of the Kubernetes metadata elements: `apiVersion`, `kind`, and `metadata`.
+
+Therefore, a string representation of the rest of the file is likely the best that can be done here.
+
+#### Fanout status update problems
+
+The fanout problem is that, when an update takes place in a single object (a policy, or an object with a policy attached), an implementation may need to update _many_ objects if it needs to place details of what policy applies, or what the resultant set of policies is on _every_ object.
+
+Historically, this is a risky strategy and needs to be carefully applied, as it's an excellent way to create apiserver load problems, which can produce a large range of bad effects for cluster stability.
+
+This does not mean that nothing at all that affects multiple objects can be done, but that careful consideration of what information is stored in status, so that _every_ policy update does not require a corresponding status update, is advised.
+
+## Current use of policies
+
+### Implementations
+
+These are a few known implementations of policies in compliance with this GEP.
+
+Users should refer to the official documentation from each implementation for for more up to date information.
+
+#### Gateway API (core)
+
+Gateway API defines two kinds of Direct policies, both for augmenting the behavior of Kubernetes `Service` resources:
+
+| Policy kind               | Description                                                                                                     | Target kinds    | Merge strategies | Policy class |
```

Comment:
We should standardize the Status API for existing policies. A Status API column is required to ensure these APIs are actually compliant with the entirety of this proposal.

Reply:
+1. It's fine if this is a follow up, but I think it's critical to at least note that this needs to be done and/or point to what we're already doing with our Gateway API policies as a recommendation for others.